.github/build.yml

```diff
@@ -0,0 +1,48 @@
+name: SmallRye Build
+
+on:
+  push:
+    branches: 
+      - main
+    paths-ignore:
+      - '.gitignore'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '.gitignore'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [
+          { 'version': '11', 'opts': '-DskipTests' },
```

Comment:
We might go with just 21. I'm sure it doesn't compile on 11 because we use records 😉